bitrise.yml

```diff
@@ -18,6 +18,15 @@ workflows:
   test:
     steps:
     - go-list: {}
+    - script:
```

Comment:
nit: another way to do this: https://github.com/bitrise-steplib/steps-check/commit/474fb5639e6e5b7f7862f38f0ea087bdc65176f4

Reply:
This is much better. 👍🏻